.github/workflows/bazel_test.yml

```diff
@@ -30,6 +35,9 @@ jobs:
             clang-10 \
             libunwind-dev \
             libblocksruntime-dev
+      - name: Use Bazel last_green on schedule
+        if: github.event.schedule == '0 0 * * *'
```

Comment:
Checking for the specific scheduling pattern is brittle. If the next person changes the schedule, this if statement will silently be dropped.

It is also unreasonable to have to configure the same scheduling pattern in all N places. This should only happen in a single place, at the beginning.

Any ideas of how to fix this?

Reply:
I haven't found any documentation on what `github.event.schedule` will evaluate to if the workflow hasn't been triggered by a schedule. It is also quite difficult to test. Furthermore, GitHub Actions doesn't support YAML anchors: https://github.community/t/support-for-yaml-anchors/16128/4.

Do you have a guess for what the default value of `github.event.schedule` should be?

Reply:
> I haven't found any documentation on what `github.event.schedule` will evaluate to if the workflow hasn't been triggered by a schedule. It is also quite difficult to test. Furthermore, GitHub Actions doesn't support YAML anchors: https://github.community/t/support-for-yaml-anchors/16128/4.
> 
> Do you have a guess for what the default value of `github.event.schedule` should be?

`null` perhaps? You can quickly test it by running a `run: exit 1` action on the condition.

Reply:
I was looking at https://docs.github.com/en/actions/learn-github-actions/expressions#literals

Reply:
Another idea: Unconditionally run `echo ${{ github.event.schedule }}` and see if it prints the value used for non-scheduled runs.

Reply:
Thanks for the idea. I tried `echo "'${{ github.event.schedule }}'"` and got `''`. I updated the PR accordingly, which gets rid of the repeated schedule. 

I haven't found a way to conditionally define an environment variable for an entire workflow though.